can/bus.py

```diff
@@ -9,7 +9,9 @@
 import logging
 import threading
 from can.broadcastmanager import ThreadBasedCyclicSendTask
+from collections import namedtuple
 
+BusState = namedtuple('BusState', 'INITIALISING, OPERATIONAL, LISTEN_ONLY, SLEEP')
```

Comment:
As Christian commented earlier - can we use ACTIVE/PASSIVE.

Perhaps we should introduce an unknown and error state too - with unkown being the default for interfaces which haven't yet implemented it.

Also it seems like you are mixing bus mode and bus state. The bus mode is up to the caller, while the state is really up to the interface/hardware.

Reply:
If we have many states, using an enum would be nice. But that would require [some library](https://pypi.python.org/pypi/enum34) for Python < 3.3.